doc/api/child_process.markdown

```diff
@@ -455,23 +180,31 @@ leaner than [`child_process.exec()`][]. It has the same options.
   * `gid` {Number} Sets the group identity of the process. (See setgid(2).)
 * Return: ChildProcess object
 
-This is a special case of the [`child_process.spawn()`][] functionality for
-spawning Node.js processes. In addition to having all the methods in a normal
-ChildProcess instance, the returned object has a communication channel built-in.
-See [`ChildProcess#send()`][] for details.
+The `child_process.fork()` method is a special case of
+[`child_process.spawn()`][] used specifically to spawn new Node.js processes.
+Like `child_process.spawn()`, a `ChildProcess` object is returned. Unlike
+`child_process.spawn()` however, the returned `ChildProcess` will have an
+additional communication channel built-in that allows messages to be passed
+back and forth between the parent and child. See [`ChildProcess#send()`][] for
+details.
 
-These child Node.js processes are still whole new instances of V8. Assume at
-least 30ms startup and 10mb memory for each new Node.js. That is, you cannot
-create many thousands of them.
+Spawned child Node.js processes include their own whole new instances of V8.
```

Comment:
v8 seems a bit of a red herring: also new instances of libc, openssl, etc., etc. Can we say something about being a completely independent Node.js process? Don't we already say that, actually?